engine/src/main/java/org/entando/entando/aps/system/services/entity/AbstractEntityTypeService.java

```diff
@@ -360,14 +356,13 @@ protected IEntityManager extractEntityManager(String entityManagerCode) {
 
     protected boolean filterObjects(Object bean, Filter[] filters, Map<String, String> mapping) {
         try {
-            if (null == filters || filters.length == 0) {
+            if (ArrayUtils.isEmpty(filters)) {
                 return true;
             }
             Map<String, String> properties = BeanUtils.describe(bean);
             for (Filter filter : filters) {
                 String fieldName = this.getFieldName(filter.getAttribute(), mapping);
-                String value = (null != properties.get(fieldName))
-                        ? properties.get(fieldName) : null;
+                String value = properties.get(fieldName);
                 if (null == value) {
                     continue;
                 }
```

Comment:
Here the filtering is ignoring the operator.
I fixed the same issues for other classes in this PR, that is still open:
https://github.com/entando/entando-core/pull/747
I can fix this in my branch after this PR has been merged.